packages/apps/staking/src/components/Header/index.tsx

```diff
@@ -0,0 +1,165 @@
+import { FC, useState } from 'react';
+import {
+  AppBar,
+  Box,
+  Link as MuiLink,
+  Toolbar,
+  IconButton,
+  Drawer,
+  Typography,
+  styled,
+  Button,
+} from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import CloseIcon from '@mui/icons-material/Close';
+import { Link } from 'react-router-dom';
+import { useAccount } from 'wagmi';
+
+import Account from '../Account';
+import ConnectWallet from '../Wallet/ConnectWallet';
+import StakeModal from '../modals/StakeModal';
+import logoImg from '../../assets/logo.svg';
+import { colorPalette } from '../../assets/styles/color-palette';
+
+const NavLink = styled(MuiLink)({
+  color: colorPalette.primary.main,
+  padding: '6px 8px',
+  fontSize: '14px',
+  lineHeight: '150%',
+  letterSpacing: '0.1px',
+  fontWeight: 600,
+  textDecoration: 'none',
+  cursor: 'pointer',
+});
+
+const DefaultHeader: FC = () => {
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
+  const [stakeModalOpen, setStakeModalOpen] = useState(false);
+  const { isConnected } = useAccount();
+
+  const toggleDrawer = (open: boolean) => {
+    setMobileMenuOpen(open);
+  };
+
+  return (
+    <AppBar
+      className="container"
+      position="static"
+      sx={{ background: '#fff', boxShadow: 'none' }}
+    >
+      <Toolbar
+        sx={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          width: '100%',
+          height: { xs: 64, md: 82 },
+        }}
+      >
+        <Link to="/" style={{ display: 'flex', alignItems: 'center' }}>
+          <img
+            src={logoImg}
+            alt="Staking Dashboard"
+            style={{ width: 118, height: 28 }}
+          />
+          <Typography
+            variant="body1"
+            ml={1.5}
+            display={{ xs: 'none', sm: 'block' }}
+          >
+            Staking
+          </Typography>
+        </Link>
+        <Box
+          display={{ xs: 'none', md: 'flex' }}
+          paddingY={{ sm: 0, md: 2.5 }}
+          height="100%"
+          alignItems="center"
+          gap={2}
+        >
+          <NavLink href="/">Staking Overview</NavLink>
```

Comment:
For app-wise routes (aka react-router), let's please use some enum or sort of `const routes = {`